Portal/src/Datahub.Portal/Datahub.Portal.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-	<Version>2.2.2</Version>
+	<Version>2.4.1</Version>
```

Comment:
can you bump to 2.6.1!

I forgot to do it and it's on my list of things to do 😄